sqlite_connector.py

```diff
@@ -147,7 +147,8 @@ def _handle_list_columns(self, param):
         query = "PRAGMA table_info({});".format(table_name)
 
         try:
-            self._cursor.execute(query)
+            # With the table existence check above, we are safe from sql injection.
+            self._cursor.execute(query)  # nosemgrep: python.lang.security.audit.formatted-sql-query.formatted-sql-query
```

Comment:
For more info on why this is safe, see https://stackoverflow.com/questions/39985599/parameter-binding-not-working-for-sqlite-pragma-table-info